gusto/physics/held_suarez_forcing.py

```diff
@@ -0,0 +1,173 @@
+import numpy as np
+from firedrake import (Interpolator, Function, dx, pi, SpatialCoordinate,
+                       split, conditional, ge, sin, dot, ln, cos, inner, Projector)
+from firedrake.fml import subject
+from gusto.core.coord_transforms import lonlatr_from_xyz
+from gusto.recovery import Recoverer, BoundaryMethod
+from gusto.physics.physics_parametrisation import PhysicsParametrisation
+from gusto.core.labels import prognostic
+from gusto.equations import thermodynamics
+from gusto.core.configuration import Held_Suarez_Parameters
+
+
+class Relaxation(PhysicsParametrisation):
+    """
+    Relaxation term for Held Suarez
+    """
+
+    def __init__(self, equation, variable_name, parameters=None):
+        """
+        Args:
+            equation (:class:`PrognosticEquationSet`): the model's equation.
+            variable_name (str): the name of the variable
+
+        """
+        label_name = f'relaxation_{variable_name}'
+        super().__init__(equation, label_name, parameters=None)
+
+        if equation.domain.on_sphere:
+            x, y, z = SpatialCoordinate(equation.domain.mesh)
+            _, lat, _ = lonlatr_from_xyz(x, y, z)
+        else:
+            # TODO: this could be determined some other way
+            # Take a mid-latitude
+            lat = pi / 4
+
+        self.X = Function(equation.X.function_space())
+        X = self.X
+        self.domain = equation.domain
+        theta_idx = equation.field_names.index('theta')
+        self.theta = X.subfunctions[theta_idx]
+        Vt = equation.domain.spaces('theta')
+        rho_idx = equation.field_names.index('rho')
+        rho = split(X)[rho_idx]
+
+        boundary_method = BoundaryMethod.extruded if equation.domain.vertical_degree == 0 else None
+        self.rho_averaged = Function(Vt)
+        self.rho_recoverer = Recoverer(rho, self.rho_averaged, boundary_method=boundary_method)
+        self.exner = Function(Vt)
+        self.exner_interpolator = Interpolator(
+            thermodynamics.exner_pressure(equation.parameters,
+                                          self.rho_averaged, self.theta), self.exner)
+        self.sigma = Function(Vt)
+        self.kappa = equation.parameters.kappa
+        HS_params = Held_Suarez_Parameters
```

Comment:
I think these parameters should be the `parameters` that are passed as an argument to the physics scheme, which means there is no need to introduce `HS_params`